operations/mimir/rollout-operator.libsonnet

```
@@ -49,6 +79,10 @@
 
   rollout_operator_node_affinity_matchers:: [],
 
+  // Create custom resource template for PodDisruptionZoneBudget
+  zpdb_crd:: std.parseYaml(importstr 'zone-aware-pod-disruption-budget-crd.yaml'),
+  zpdb_custom_resource: if !zpdb_resources_enabled then null else $.zpdb_crd,
```

[nit] Might be able to simplify this a little:

```suggestion
  zpdb_custom_resource: if !zpdb_resources_enabled then null else std.parseYaml(importstr 'zone-aware-pod-disruption-budget-crd.yaml'),
```

---

I just did this to match the way the replica template CRD was done. I'll adjust both to your suggestion as I pull this jsonnet into the rollout operator repo.